### DISCUSSION TITLE
Unable to retrieve the file_size for file at location: livewire-tmp/livewire-tmp. For files more than 100mb

### DISCUSSION THREAD

**Discussion by nikorobos** [filamentphp/filament#12073] [Help]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Hello i am trying to upload a file that is more than 100mb but it fails. Even though it shows 
![image](https://github.com/filamentphp/filament/assets/30929830/675e17f0-f6a1-4261-8c6c-8bf07326e595)

When i click submit then i get the error 
`Unable to retrieve the file_size for file at location: livewire-tmp/livewire-tmp.`

I have xampp and in php.ini i have set 
`upload_max_filesize=200M`
`post_max_size=200M`
`memory_limit=512M`
`max_execution_time=1800`
`max_input_time=1800`

i have published the livewire.php and inside here i have 

```
'temporary_file_upload' => [
        'disk' => null,        // Example: 'local', 's3'              | Default: 'default'
        'rules' => 'max:2048000',     // Example: ['file', 'mimes:png,jpg']  | Default: ['required', 'file', 'max:12288'] (12MB)
        'directory' => null,   // Example: 'tmp'                      | Default: 'livewire-tmp'
        'middleware' => null,  // Example: 'throttle:5,1'             | Default: 'throttle:60,1'
        'preview_mimes' => [   // Supported file types for temporary pre-signed file URLs...
            'png', 'gif', 'bmp', 'svg', 'wav', 'mp4',
            'mov', 'avi', 'wmv', 'mp3', 'm4a',
            'jpg', 'jpeg', 'mpga', 'webp', 'wma',
        ],
        'max_upload_time' => 30, // Max duration (in minutes) before an upload is invalidated...
    ],
```
    
And in my filament resource page on the form i have 
    
```
    Forms\Components\FileUpload::make('file_path')
                    ->label('Αρχείο')
                    ->required()
                    ->disk('app_versions')
                    ->preserveFilenames()
                    ->visibility("public")
                    ->maxSize(2048000),
```           
If i upload every other file that is less than 100mb it is uploaded without any problem. 
    
Is there something else that i need to change on the php.ini or in livewire.php so that the upload works for files more than 100mb?

**Comment by Mahdi-Sahib** [comment]
same Issues, How did you solve this problem?

**Comment by mbh00** [comment]
have you already executed the command `php artisan storage:link`

**Comment by khoait03** [comment]
Have you executed the php artisan storage:link command?
and check file .env APP_URL

If so, check to see if there is a public/storage directory
If there is a public/stogare folder but you cannot upload photos yet.
Delete the public/stogare folder.
And run the php artisan storage:link command again
